db/schema.rb

```diff
@@ -1433,7 +1433,6 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
     t.index ["twitter_username"], name: "index_users_on_twitter_username", unique: true
     t.index ["username"], name: "index_users_on_username", unique: true
-    t.check_constraint "username IS NOT NULL", name: "users_username_not_null"
```

Comment:
I cant think HOW this got deleted 😖  I'm open to suggestions on correcting

Reply:
For the purpose of fixing this PR, you can just add the line back in the same spot.